card/mod/06_bootstrap/set/self/bootstrap_js.rb

```diff
@@ -1,5 +1,7 @@
 view :raw do |args|
-  File.read "#{Cardio.gem_root}/mod/06_bootstrap/lib/javascript/bootstrap.js"
+  [ "bootstrap.js", "bootstrap_modal_wagn.js" ].map do |name|
+    File.read "#{Cardio.gem_root}/mod/06_bootstrap/lib/javascript/#{name}"
+  end.join("\n")
```

Comment:
seems like we should keep the bootstrap_js pure and add this handling to the wagn_mod js. (The idea has been that we will eventually break out the wagn_mod stuff into different mods)